src/ui/public/fixed_scroll.js

```diff
@@ -98,16 +103,28 @@ uiModules
         listen();
       }
 
-      // reset when the width or scrollWidth of the $el changes
-      $scope.$watchMulti([
-        function () { return $el.prop('scrollWidth'); },
-        function () { return $el.width(); }
-      ], setup);
+      let width;
+      let scrollWidth;
+      let widthCheckTimeout = $timeout(checkWidth, 500);
```

Comment:
Hmm, is there anything we can listen to (like window resize) to invoke this, or do we really have to do it every half-second?